lib/bundler/source/path/installer.rb

```diff
@@ -11,8 +11,8 @@ def initialize(spec, options = {})
           @gem_bin_dir       = "#{Bundler.rubygems.gem_dir}/bin"
           @bin_dir           = Bundler.requires_sudo? ? @tmp_bin_dir : @gem_bin_dir
           @gem_dir           = Bundler.rubygems.path(spec.full_gem_path)
-          @wrappers          = options[:wrappers] || true
-          @env_shebang       = options[:env_shebang] || true
+          @wrappers          = true
```

Comment:
Given that it changed 5 years ago in 93033523 and no one noticed, I think we should probably keep it the way it is now.